priv/cacerts.pem

```diff
@@ -0,0 +1,4606 @@
+
```

Comment:
Not sure about the name or location. But whatever changes there'll have to be changes to the vendoring script, too.